src/_gettsim/transfers/kindergeld.py

```diff
@@ -11,20 +11,25 @@
     },
 }
 
+aggregate_by_p_id_kindergeld = {
+    "kindergeld_anz_ansprüche": {
+        "p_id_to_aggregate_by": "p_id_kindergeld_empf",
+        "source_col": "kindergeld_anspruch",
+        "aggr": "sum",
+    },
+}
+
 
 def kindergeld_m(
```

Comment:
Should be simplified to

```python
kindergeld_params["kindergeld"] * kindergeld_anz_ansprüche
```

starting in 2023. Either do here or make an issue.